src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -73,8 +72,10 @@ public function __construct(array $mockedNamespaces = array())
                 $enableDebugClassLoader = $namespaces && $namespaces[0];
```

Comment:
Now there are two DebugClassLoader, how can you deal with this scenario specifically?